modules/View.js

```diff
@@ -441,6 +475,55 @@
 
     /************** Private methods **************/
 
+
+    /**
+    * sends event triggers to Torso.Events
+    * @param array of timing event names
+    * @method applyTimingEventListeners
+    */
+    __applyTimingEventListeners: function(timingEvents) {
```

Comment:
Not a big fan of having this repeated in every module. Can this be in whatever "timing events" module you create after speaking to @mandragorn and @arikwex?